samples/cellular/modem_shell/thingy91x_wifi.overlay

```diff
@@ -0,0 +1,6 @@
+/* Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <thingy91x_wifi.dtsi>
```

Comment:
Hmmm, could we actually have this always included (by being under boards folder) even if WiFi would be disabled by default?
I guess it shouldn't cause any power consumption penalty etc. if WiFi is not turned on by CONFIG_WIFI etc. Kconfig symbols?

Reply:
I tried building the sample with and without the Wi-Fi include. There's a small (less than 200 bytes) difference in flash consumption. The increased flash consumption is not a problem as such, but I'm worried if the DTSI may affect current consumption even without the driver being enabled.

Reply:
We decided to keep it like it's now, just in case.